.editorconfig

```diff
@@ -0,0 +1,6 @@
+[*]
```

Comment:
This file should be excluded

Reply:
its a high-level configuration of tab/space formatting for many IDEs. for RN its quite common that devs use many IDEs: Intellij, vscode, atom, etc. .editorconfig is the only cross-ide thing that recognized by all of them.